server/go/main.go

```diff
@@ -0,0 +1,95 @@
+//go:generate protoc -I ../protos --go_out=plugins=grpc:. ../protos/appencryption.proto
+
+// Package main implements a gRPC server that ...
+package main
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"net"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/jessevdk/go-flags"
+	"google.golang.org/grpc"
+
+	pb "github.com/godaddy/asherah/server/go/api"
+	"github.com/godaddy/asherah/server/go/pkg/server"
+)
+
+type Options struct {
+	SocketFile flags.Filename  `short:"s" long:"socket-file" default:"/tmp/appencryption.sock" description:"The unix domain socket the server will listen on"`
+	Asherah    *server.Options `group:"Asherah Options"`
+}
+
+func main() {
+	opts := new(Options)
+	parser := flags.NewParser(opts, flags.Default)
+
+	if _, err := parser.Parse(); err != nil {
+		if e, ok := err.(*flags.Error); ok && e.Type == flags.ErrHelp {
+			return
+		}
+
+		parser.WriteHelp(os.Stdout)
+
+		return
+	}
+
+	if err := validateOptions(opts); err != nil {
+		fmt.Println("Missing or invalid options:", err.Error())
+		fmt.Println()
```

Comment:
can possibly replace this with `fmt.Printf("Missing or invalid options: %s\n", err.Error())`?

Reply:
Two newlines (one blank line) is intentional. Replacing with the `fmt.Printf` example would eliminate the blank line. Of course, we could use `fmt.Printf` with `\n\n` but I tend to find `fmt.Println()` to be more explicit/readable.